src/core/api/chart_repository.go

```diff
@@ -555,3 +622,24 @@ func chartFullName(namespace, chartName, version string) string {
 	}
 	return fmt.Sprintf("%s/%s:%s", namespace, chartName, version)
 }
+
+// parseChartVersionFromFilename parse chart and version from file name
+func parseChartVersionFromFilename(filename string) (string, string) {
```

Comment:
@steven-zou please confirm whether we can use this func to get version and name?